managed_headers.go

```diff
@@ -0,0 +1,64 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+type ListManagedHeadersResponse struct {
+	Response
+	Result ManagedHeaders `json:"result"`
+}
+
+type ManagedHeaders struct {
+	ManagedRequestHeaders  []ManagedHeader `json:"managed_request_headers"`
+	ManagedResponseHeaders []ManagedHeader `json:"managed_response_headers"`
+}
+
+type ManagedHeader struct {
+	ID            string   `json:"id"`
+	Enabled       bool     `json:"enabled"`
+	HasCoflict    bool     `json:"has_conflict,omitempty"`
+	ConflictsWith []string `json:"conflicts_with,omitempty"`
+}
+
+func (api *API) ListZoneManagedHeaders(ctx context.Context, zoneID string) (ManagedHeaders, error) {
```

Comment:
instead of only accepting zoneID, can we make it accept a struct (with the single param) as a bit of future proofing and consistency with the newer approach?

Reply:
Makes sense, fixed it